.ci/jenkins_job_build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+vc_repo="github.com/containers/virtcontainers"
+
+# Export all environment variables needed.
+export ghprbPullId
+export ghprbTargetBranch
```

Comment:
Both `ghprbPullId` and `ghprbTargetBranch` need to be exported from the Jenkins job directly.